internal/driver/fetch.go

```diff
@@ -529,16 +529,23 @@ func convertPerfData(perfPath string, ui plugin.UI) (*os.File, error) {
 	ui.Print(fmt.Sprintf(
 		"Converting %s to a profile.proto... (May take a few minutes)",
 		perfPath))
-	profile, err := newTempFile(os.TempDir(), "pprof_", ".pb.gz")
+	// Get a tempFile
+	tempFilename, err := getNextTempFile(os.TempDir(), "pprof_", ".pb.gz")
 	if err != nil {
 		return nil, err
 	}
-	deferDeleteTempFile(profile.Name())
-	cmd := exec.Command("perf_to_profile", perfPath, profile.Name())
+	cmd := exec.Command("perf_to_profile", perfPath, tempFilename)
```

Comment:
By the way, I think the perf_data_converter change that broke its usage by pprof is this commit: https://github.com/google/perf_data_converter/pull/8/files#diff-183b816d357be307fcb59fc0a2fd2e03. Before that commit, perf_to_profile would happily overwrite the destination file. With that change, it started to refuse the conversion if the destination file exists. I know wonder what was the reason for that change. Without that change no pprof changes would be needed.